File: Semester 1/Fundamentals of programming/Labs/a8/src/domain/Student.py

class Student:

    def __init__(self, student_id, name,):
        self.__name = name
        self.__student_id = student_id

    @property
    def student_id(self):
        return self.__student_id

    @property
    def name(self):
        return self

    def set_name(self, name):
        self.__name = name


    def __str__(self):
        return f"{self.__student_id} - {self.__name}"

File: Semester 3/Semester 3 Ro/Metode_Avansate_de_Programare/cursuri/exemple_curs4_5/curs4/src/main/java/domain/Student.java
package domain;

public class Student extends Entity<Long>{
    private String name;
    private int group;

    public Student() {
    }

    public Student(String name, int group) {
        this.name = name;
        this.group = group;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getGroup() {
        return group;
    }

    public void setGroup(int group) {
        this.group = group;
    }

    @Override
    public String toString() {
        return "Student{" + getId().toString()+
                " name='" + name + '\'' +
                ", group=" + group +
                '}';
    }
}
